src/components/atoms/Radio/Radio.tsx

```diff
@@ -0,0 +1,182 @@
+import HelperText from "../../atoms/HelperText";
+
+// Styles
+import "./Radio.scss";
+
+interface IRadioProps {
+    /**
+     *  The text displayed as the label for the radio, describing its purpose or function.
+     */
+    label?: string;
+    /**
+     *  Toggles the label's and HelperText position between above or beside the radio.
+     */
+    vertical?: boolean;
+    /**
+     *  Specifies whether the radio is mandatory for completing a form.
+     */
+    required?: boolean;
+    /**
+     *  Disables the radio, preventing it from being interacted with.
+     */
+    disabled?: boolean;
+    /**
+     *  Displays the radio as read-only, where users cannot modify its value.
+     */
+    readOnly?: boolean;
+    /**
+     *  Manages the checked state of the radio in a controlled way.
+     */
+    checked?: boolean;
+    /**
+     *  Automatically focuses the radio when the component mounts.
+     */
+    autoFocus?: boolean;
+    /**
+     *  Extra information displayed with the label for clarity or guidance.
+     */
+    infoText?: string;
+    /**
+     *  Helper text to provide context or explain any errors, warnings related to the radio.
+     */
+    helperText?: string;
+    /**
+     *  The initial checked state of the radio before user interaction.
+     */
+    defaultChecked?: boolean;
+    /**
+     *  Determines the radioes appearance based on its status.<br>
+     *  Possible values: `rest | warning | error`
+     */
+    type?: "rest" | "warning" | "error";
+    /**
+     *  HTML name attribute for the input element.<br>
+     *  A unique identifier for the radio within a form.
+     */
+    name: string;
+    /**
+     *  Fires when the user changes the radio state. Provides the change event as a callback's argument.
+     */
+    onChange?: (e: ChangeEvent<HTMLInputElement>) => void;
+    /**
+     *  Event handler for when the radio input element loses focus. Provides the focus event as a callback's argument.
+     */
+    onBlur?: (e: FocusEvent<HTMLInputElement>) => void;
+    /**
+     *  Event handler for when the radio input element receives focus. Provides the focus event as a callback's argument.
+     */
+    onFocus?: (e: FocusEvent<HTMLInputElement>) => void;
+    /**
+     * Additional class for the parent element.
+     * This prop should be used to set placement properties for the element relative to its parent using BEM conventions.
+     */
+    className?: string;
+    /**
+     * The value that returns in the onChange event
+     * */
+    value: string;
+}
+
+/**
+ * radio component allows users to select one or more options from a set of choices. Each radio can be either checked or unchecked, indicating a binary state. radioes are commonly used in forms, settings, and lists where multiple selections are needed.
+ */
+const Radio: FC<IRadioProps> = ({
+    label,
+    required,
+    infoText,
+    disabled,
+    helperText,
+    readOnly,
+    type = "rest" as const,
```

Comment:
Please replace the code to 

```suggestion
    type = "rest",
```